src/main/gui/splash.ts

```diff
@@ -62,12 +62,16 @@ export const addSplashScreen = (monitor: ProgressMonitor) => {
     }, 100);
 
     splash.once('ready-to-show', () => {
-        splash.show();
+        if (!splash.isDestroyed()) {
+            splash.show();
+        }
     });
 
     monitor.addProgressListener((progress) => {
         lastProgress = { ...progress };
-        splash.webContents.send('splash-setup-progress', progress);
+        if (!splash.isDestroyed()) {
+            splash.webContents.send('splash-setup-progress', progress);
+        }
```

Comment:
This one can be removed. There's already an interval constantly sending `lastProgress` to the splash. Adding this in the first place was a mistake on my part.

```suggestion
```